cmd/lock-rpc-server.go

```diff
@@ -76,11 +76,12 @@ func startLockMaintenance(lockServers []*lockServer) {
 			for {
 				// Verifies every minute for locks held more than 2minutes.
 				select {
-				case <-ticker.C:
-					lk.lockMaintenance(lockValidityCheckInterval)
 				case <-globalServiceDoneCh:
-					// Stop the timer.
+					// Stop the timer upon service closure and cleanup the go-routine.
 					ticker.Stop()
+					return
+				case <-ticker.C:
+					lk.lockMaintenance(lockValidityCheckInterval)
```

Comment:
why did the ticker.c channel order need to be swapped in the select - it wouldn't matter, would it?